app/assets/stylesheets/orchid.scss

```diff
@@ -33,6 +33,23 @@
 
 // Site Section-specific
 // =====================
+.site_browse {
+  .browse-item {
+    border-bottom: dotted 0.0125em;
```

Comment:
Is specifying borders in ems a bootstrap convention? It seems like it has the potential for weird results (over pixels). I don't think this should stop a merge, just curious.

Reply:
No, just another of my preferences. I write everything in ems. This used to be necessary for relative sizing with zooming but that is a holdover from when browsers wouldn't resize things defined in pixels. Otherwise, I just like to be consistent and decided sticking with ems is easier to figure out than kind of arbitrarily using px in some places and ems in others.

Reply:
I think moving some of those sizings into Sass variables could help simplify writing them, but I've gotten used to translating pixel sizes into ems